### ISSUE TITLE
Battery level not reported correctly

### ISSUE THREAD

**Issue by MrWGT** [SeydX/homebridge-fritz-platform#118]
I use the following config:

{
          "accType" : "thermostat",
          "ain" : "099950409035",
          "battery" : true,
          "name" : "Heizung Leana",
          "temperature" : true,
          "window" : false
        },
        {
          "accType" : "window",
          "ain" : "099950409035",
          "battery" : true,
          "name" : "Fenstererkennung KiZi"
        },

Same AIN but battery is reported correct as Thermostat but not as window.

Battery level is also wrong on HAN-FUN optical window contact:

{
          "accType" : "contact",
          "ain" : "119340208370",
          "battery" : true,
          "name" : "Kleines Fenster KiZi"
        },

**Comment by SeydX** [comment]
@MrWGT 

Thy i will check this

**Comment by SeydX** [comment]
@MrWGT 

I can not reproduce this atm. Need to find a empty battery to check it 😅

**Comment by MrWGT** [comment]
> @MrWGT 
> 
> 
> 
> I can not reproduce this atm. Need to find a empty battery to check it 😅

The battery is not empty. It's just shown as empty. 😅

**Comment by MrWGT** [comment]
The thermostat and the window contact have the same AIN. Thermostat is configured as window = false. Window contact (from Thermostat) as separate device shows empty battery. But thermostat not.

**Comment by MrWGT** [comment]
[2020-11-15 16:35:30] The plugin "homebridge-fritz-platform" requires Node version of >=12.18.3 which does not satisfy the current Node version of v12.16.1. You may need to upgrade your installation of Node.

Could this be the reason?

**Comment by SeydX** [comment]
@MrWGT 

No node is not causing that, i will check the batter again

**Comment by SeydX** [comment]
@MrWGT 

Hat dein Kontaktsensor auch eine Batterieanzeige auf der FritzBox Oberfläche?

**Comment by MrWGT** [comment]
Ja, es ist ein HAN-FUN Sensor von der Telekom. Dieser wird von der Fritzbox unterstützt.

Anzeige in der Oberfläche "Batterie OK". Keine Prozent.

**Comment by MrWGT** [comment]
Bilder von Eve und Fritzbox hier:

https://1drv.ms/f/s!Assv5fIy-dXzgpdU1Kq0LoS_mfrySA

**Comment by SeydX** [comment]
@MrWGT 

Window habe ich gefixt, kommt mit der nächsten Version. Bei den Kontaktsensoren ist es etwas schwieriger. Die Batteri zeigt keine Prozentzahl an und ich weiß nicht woher die FritzBox Anzeige den Status "OK" her hat weil es nicht in der API definiert ist.

**Comment by MrWGT** [comment]
In der vorherigen Version des Plugins ging es. Es wurde kein Fehler angezeigt.

**Comment by SeydX** [comment]
@MrWGT 

Wurde die richtige Prozentzahl angezeigt beim Kontaktsensor?

**Comment by MrWGT** [comment]
Hm... Weiß ich leider nicht mehr. Es war zumindest keine Warnung "Batterie leer" da.

**Comment by SeydX** [comment]
@MrWGT 

das liegt daran weil die alte Version überhaupt keine Batterianzeige hatte. 

v5.0.12 ist online. Bitte einmal updaten und den Eintrag für Batterie im config für den Kontaktsensor entfernen oder auf false setzen. Anschließen hb neustarten. Sollte fixen.

**Comment by MrWGT** [comment]
Habe ich gemacht und funktioniert jetzt. Nur der echte Fensterkontkt bleibt noch übrig 😅

Danke 😀

**Comment by SeydX** [comment]
@MrWGT 


Der echte Fensterkontakt?

**Comment by MrWGT** [comment]
Ja. Derjenige welcher keine Prozent anzeigt.

https://www.smarthome.de/geraete/smarthome-tuer-fensterkontakt-optisch-weiss

Den habe ich bei Ebay für 15€ neu bekommen.

Auf und Zu werden sicher und schnell erkannt. Fritzbox 7590 zeigt den als HAN-FUN an und Batterie als OK ohne Prozent

**Comment by MrWGT** [comment]
Habe mal das Debug-Log angeschalten:

16.11.2020, 22:27:25 - [FritzPlatform] Kleines Fenster KiZi: Polling state (smarthome-window)...
16.11.2020, 22:27:25 - [FritzPlatform] Kleines Fenster KiZi: {"name":"Kleines Fenster KiZi","ain":"119340208370-1","online":1,"bitmask":"8208","busy":0,"battery":false,"alert":{"state":0},"temperature":false,"powermeter":false,"switch":false,"button":false,"thermostat":false,"light":false}

Hilft das was?

**Comment by SeydX** [comment]
@MrWGT 

Gut gemeint, leider nicht. Da die Anzeige da von mir manipuliert ist und wie du siehst steht da "battery": false - was aber nicht heißen muss das die API keine Informationen dazu hergibt

Müsste ein kleines debug modul entwerfen für dich was ALLE API daten ohne manipulation anzeigt. Villeicht finden wir da was brauchbares
